debugging/coreclr/dockerClient.ts

```diff
@@ -64,6 +69,14 @@ export class CliDockerClient implements DockerClient {
             command += ` -f ${options.dockerfile}`;
         }
 
+        if (options && options.args) {
+            command += Object.keys(options.args).map(arg => ` --build-arg "${arg}=${options.args[arg]}"`).join('');
```

Comment:
There's gotta be a cleaner way to add all of these arguments. Maybe instead of appending them to the command as you go, you do something like this:

```typescript
let command = 'docker build --rm';
let commandArgs: string[] = [];
options = options || {}; // so that you don't have to check `if (options)` every time

commandArgs.push(...formatKeyValArg('label', options.labels));

command += commandArgs.join(' ');

function formatKeyValArg(argName: string, values?: { [key: string]: string } | undefined): string[] {
    if (values) {
        return Object.keys(values).map(key => `--${argName} "${key}=${values[key]}"`);
    }
}
```

I'm mainly suggesting this to avoid a possible bug due to a missing/misplaced quote or space.